app/javascript/src/Policies/actions/PolicyChain.jsx

```diff
@@ -2,10 +2,10 @@
 
 import { RSAA } from 'redux-api-middleware'
 
-import type { RSSAAction, RegistryPolicy, ChainPolicy, StoredChainPolicy } from 'Policies/types'
+import type { RSSAAction, IPolicy, ChainPolicy, PolicyConfig } from 'Policies/types'
```

Comment:
Just a question about naming. Why `IPolicy` is better than `RegistryPolicy`? Same for `PolicyConfig`.
What does the `I` means?